docs/components/Changelog.js

```diff
@@ -8,6 +8,11 @@ class Changelog extends React.Component {
 
         <h2>MX React Components V 6.1.0</h2>
 
+          <h3>6.1.9</h3>
```

Comment:
Looks like this version is now wrong because Drew merged and released while we were reviewing this. I think this will now need to be 6.1.11 but I would get with @DrewWarrenTIY  and make sure you guys have the change log updated correctly between the two of you. He'll have another MR that will probably go along with yours on this release.